src/theme/components/subnav.ts

```diff
@@ -0,0 +1,207 @@
+  fontWeight: "regular",
+  gap: "xs",
+  height: { base: "44px", md: "unset" },
+  lineHeight: "1.5 !important",
+  position: "relative",
+  px: "s",
+  py: "xxs",
+  textDecoration: "none !important",
+  transition: "background-color 0.2s, color 0.2s !important",
+});
+
+const SubNav = subNavDefineMultiStyleConfig({
+  baseStyle: subNavDefinePartsStyle(
+    ({ backgroundColor, highlightColor }: SubNavStyleProps) => {
+      const defaultLabelColor = "ui.typography.body";
+      const highlightOrDefaultColor = highlightColor
+        ? `${highlightColor} !important`
+        : `${defaultLabelColor} !important`;
+      const highlightOrLinkColor = highlightColor
+        ? `${highlightColor} !important`
+        : "ui.link.primary !important";
+      const highlightOrBorderColor = highlightColor
+        ? `${highlightColor}`
+        : "ui.border.default";
+      const finalBackgroundColor = backgroundColor
+        ? backgroundColor
+        : "ui.link.primary-05 !important";
+      const primaryActionsStyles = {
+        ...commonStyles(),
+        svg: {
+          fill: defaultLabelColor,
+          margin: { base: "0", md: null },
+          _dark: {
+            fill: "ui.white",
+          },
+        },
+        _hover: {
+          backgroundColor: finalBackgroundColor,
+          color: highlightOrDefaultColor,
+          svg: {
+            fill: highlightOrDefaultColor,
+            _dark: {
+              fill:
+                backgroundColor !== undefined
+                  ? `${backgroundColor} `
+                  : "ui.white",
+            },
+          },
+        },
+      };
+      const secondaryActionsStyles = {
+        ...commonStyles(),
+        color: highlightOrLinkColor,
+        svg: {
+          fill: highlightOrLinkColor,
+          margin: { base: "0", md: null },
+          _dark: {
+            fill: backgroundColor
+              ? `${backgroundColor} !important`
+              : "dark.ui.link.primary-05 !important",
+          },
+        },
+        _hover: {
+          background: finalBackgroundColor,
+          svg: {
+            fill: highlightOrLinkColor,
+            _dark: {
+              fill: backgroundColor
+                ? `${backgroundColor}`
+                : "dark.ui.link.primary-05",
+            },
+          },
+        },
+      };
+      return {
+        base: {
+          ".selectedItem": {
+            color: highlightOrLinkColor,
+            fontWeight: "bold",
+            backgroundColor: finalBackgroundColor,
+            "&:hover": {
+              color: highlightOrLinkColor,
+            },
+          },
+          borderBottom: "1px solid",
+          borderColor: highlightOrBorderColor,
+          display: "flex",
+          justifyContent: "center",
+        },
+        container: {
+          maxWidth: "1280px",
+          // px: "s",
+          px: { base: "0", md: "xs" },
+          width: "100%",
+        },
+        listCleanup: {
```

Comment:
You can refactor and update this. There are two ways I'm thinking but you can update this if you think of another way.

1. You are using `...styles.listCleanup` in both `<List ...>` components. Instead of having to pass the styles in this manner, you can rename `listCleanup` as `ul`. This will target all lists in the component, but there are only two so this is ok.

2. You can set these rules in an object and add them to `primaryActions` and `secondaryActions`.

```
const ulStyles = {
  p: { base: "s", md: "xs" },
  li: {
    marginEnd: "unset",
  },
  margin: "0 !important",
};

// ...

primaryActions: {
  ...ulStyles,
  width: "100%",
  button: {
// etc

secondaryActions: {
  ...ulStyles,
  width: "fit-content",
  whiteSpace: "nowrap",
// etc

Reply:
Note, I moved the rule from line 181 inside `ulStyles` as an example.